src/drt/src/db/tech/frConstraint.h

```diff
@@ -80,12 +80,152 @@ class frConstraint
   void setLayer(frLayer* layer) { layer_ = layer; }
   void setId(int in) { id_ = in; }
   int getId() const { return id_; }
+  std::string getViolName() const
+  {
+    switch (typeId()) {
+      case frConstraintTypeEnum::frcShortConstraint:
+        return "Short";
+      case frConstraintTypeEnum::frcMinWidthConstraint:
+        return "MinWid";
```

Comment:
Since we are afforded 15 characters lets pick some better names.  Would you take a first pass through and pick nicer names.  For example

MinWid -> Min Width 
CutSpc -> Cut Spacing